src/test/java/com/tngtech/java/junit/dataprovider/internal/convert/StringConverterTest.java

```diff
@@ -385,7 +385,7 @@ public void testConvertShouldParseNullValuesAsNullObjectIfSettingsConvertNullsIs
         Object[] result = underTest.convert(data, false, parameterTypes, dataProvider, 71);
 
         // Then:
-        assertThat(result).isEqualTo(new Object[] { null, null, "foo" });
+        assertThat(result).containsExactly(null, null, "foo");
```

Comment:
here, but maybe you have a reason why this does not matter or why my gut feeling maybe betray me ...
I am open minded and this at least looks shorter ...